lib/versioning/ivy/index.js

```diff
@@ -0,0 +1,73 @@
+const {
+  equals,
+  getMajor,
+  getMinor,
+  getPatch,
+  isGreaterThan,
+  isSingleVersion,
+  isStable,
+  matches: mavenMatches,
+  maxSatisfyingVersion,
+  minSatisfyingVersion,
+  getNewValue,
+  sortVersions,
```

Comment:
Not sure about latest 4 functions reuse

Reply:
I thought we wouldn't be able to directly reuse any of the functions due to the change in syntax? e.g.

```
[1.0,2.0[ matches all versions greater or equal to 1.0 and lower than 2.0
]1.0,2.0] matches all versions greater than 1.0 and lower or equal to 2.0
]1.0,2.0[ matches all versions greater than 1.0 and lower than 2.0
]1.0,) matches all versions greater than 1.0
(,2.0[ matches all versions lower than 2.0
```